extensions/amp-subscriptions/0.1/entitlement.js

```diff
@@ -75,6 +78,7 @@ export class Entitlement {
       'granted': this.granted,
       'grantReason': this.grantReason,
       'data': this.data,
+      'decryptedDocumentKey': this.decryptedDocumentKey,
```

Comment:
Hmm. I don't think we can put it in JSON here - it's a bit of a leak liability. It's one of those things: we accept it in `fromJson`, but can't include in the `toJSON`. This is a good point to assure in a unit test.